modules/ROOT/pages/_partials/anypoint-security-policies-prereqs.adoc

```diff
@@ -5,8 +5,9 @@
 
 To configure and use the security policies, you must:
 
-. Have permission to manage policies in API Manager.
-+ 
-See xref:api-manager::environment-permission-task.adoc[Granting Permission to Access an Environment]
-. xref:runtime-fabric::install-create-rtf-arm.adoc[Install Runtime Fabric]. Anypoint Runtime Fabric is a container service that automates the deployment and orchestration of Mule apps and API gateways.
-. xref:runtime-fabric::enable-inbound-traffic.adoc[Enable inbound traffic] on Runtime Fabric to allow Mule apps and API gateways to listen on inbound connections.
\ No newline at end of file
+* Have the Anypoint Security - Edge entitlement for your Anypoint Platform account. If you don't see *Security* in *Management Center*, contact your customer success manager to enable Anypoint Security for your account.
+* Have Runtime Fabric with inbound traffic configured. +
+Anypoint Runtime Fabric is a container service that automates the deployment and orchestration of Mule apps and API gateways. +
+See xref:runtime-fabric::index.adoc[the Runtime Fabric documentation]. +
+Runtime Fabric requires an Anypoint Platform Platinum or above subscription.
```

Comment:
"Platinum or above"? If there are not too many "above," consider listing. Otherwise, this is my best effort:
```suggestion
Runtime Fabric requires an Anypoint Platform Platinum or higher-level subscription.
```